apps/judicial-system/web/src/routes/Defender/Cases/components/DefenderCasesTable.tsx

```diff
@@ -129,9 +132,13 @@ export const DefenderCasesTable: FC<Props> = ({
                 </th>
               ) : (
                 <th>
-                  <Text fontWeight="regular">
-                    {formatMessage(tables.hearingArrangementDate)}
-                  </Text>
+                  <SortButton
```

Comment:
Nit: I saw in `apps/judicial-system/web/src/components/Table/CaseFileTable/CaseFileTable.tsx`, that we created a local helper to set these props within the same component, but this could be shared accross few components now.

Reply:
We have a independant Table component that has the ability to sort things. I think we should use that here.

Reply:
Makes sense, I created another ticket for that as I'm working on another bigger task at the moment and I don't necessarily think this needs to wait as it works and is a better experience for the user. Moving it to the new table component is more for our own benefit as devs, so I don't want the change to be delayed by that 🤓